### PR TITLE
Release 1.2.1

### PR DESCRIPTION
**Pull request by rs028**
This prepares Release v1.2.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # AtChem2 - CHANGELOG
 
+## v1.2.1 (January 2021)
+
+- new environment variable `ASA` (aerosol surface area)
+- fix bug in the C-binding for `gfortran` version 9 and above
```
Comment (rs028) [inline CHANGELOG.md:6]: @spco is this correct?

Comment (spco) [reply CHANGELOG.md:6]: Looks good to me.

**Comment by rs028** [comment]
Great, I need to recompile the manual but LaTeX is not cooperating tonight. I will merge once it's done.

I guess Travis is already gone? What about Codecov?

**Comment by spco** [comment]
Codecov seems to still be functioning - it's not related directly to Travis.

**Comment by rs028** [comment]
So why it is not giving a report on this PR?

**Comment by rs028** [comment]
Never mind, here it is :)

**Comment by spco** [comment]
Very slow! :)

**Review by spco — APPROVED** [review]
All good!

**Pull request merged**